Libraries/oneDAL/IntelPython_daal4py_Distributed_Kmeans/README.md

```diff
@@ -1,11 +1,11 @@
-# daal4py Distributed K-Means 
```

Comment:
Old styleReadme, please update using the [mandelbrot readme](https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot/README.md) as an example: In this case a "Purpose" and "License" sections are missing at a minimum